Packs/ApiModules/Scripts/ServiceNowApiModule/ServiceNowApiModule_test.py

```diff
@@ -0,0 +1,62 @@
+from ServiceNowApiModule import *
+import demistomock as demisto
+
```

Comment:
I added tests for this case in the ServiceNowv2_test.py file, as the error is raised from there so it will be a specific error for each integration and not a general one.